zkstack_cli/crates/zkstack/src/commands/external_node/build.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Context;
+use common::{cmd::Cmd, logger};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use crate::messages::{MSG_BUILDING_EN, MSG_CHAIN_NOT_FOUND_ERR, MSG_FAILED_TO_BUILD_EN_ERR};
+
+pub(crate) async fn build(shell: &Shell) -> anyhow::Result<()> {
```

Comment:
No objections, but looks a bit out of place in this PR.
Combined with changes in autocomplete this has blown up the diff size.

Reply:
Makes sense; I'll make these changes in a separate PR. The reasoning behind them was to prevent data races between contract verifier compilation and the following testing logic (this applies to ENs as well; I've observed a couple of such data races recently).